train.py

```diff
@@ -168,7 +189,7 @@ def run_experiment(cfg: DictConfig, classnames: list[str]) -> nn.Module:
             "epoch": cfg.train.num_epochs,
             "model_state_dict": model.state_dict(),
             "optimizer_state_dict": optimizer.state_dict(),
-            "loss": cfg.train.loss,
+            "lr": cfg.train.lr,
```

Comment:
```suggestion
            "config": OmegaConf.to_container(cfg, resolve=True),
```

It's good practice to save the full config.